src/extractor/edge_based_graph_factory.cpp

```diff
@@ -344,13 +346,19 @@ void EdgeBasedGraphFactory::GenerateEdgeExpandedEdges(
                                          name_table,
                                          street_name_suffix_table,
                                          profile_properties);
-    guidance::lanes::TurnLaneHandler turn_lane_handler(
-        *m_node_based_graph, turn_lane_offsets, turn_lane_masks, turn_analysis);
+
+    guidance::LaneDataIdMap lane_data_map;
+    guidance::lanes::TurnLaneHandler turn_lane_handler(*m_node_based_graph,
```

Comment:
The turn lane handler here isn't const anyway and I think this is the only place we're using it. Therefore the non-const function issue above should be fine.